Formula/d/dhall-lsp-server.rb

```diff
@@ -41,22 +26,6 @@ class DhallLspServer < Formula
   uses_from_macos "zlib"
 
   def install
-    args = []
-    if build.head?
-      cd name
```

Comment:
This line still might be needed to install the formula from HEAD

Reply:
yeah, that is right 👍